setup.py

```diff
@@ -1,55 +1,51 @@
-import os
-import sys
-from setuptools import setup
+from pathlib import Path
 
+from setuptools import setup, find_packages
```

Comment:
Don't we need to install setuptools, too?

Reply:
It comes with almost all python setups, which is why it works everywhere without manually installing it but:
* Specifying it in `setup.py` isn't possible because, well `setuptools` is needed to run `setup(...)`
* Putting it in a requirements file does nothing unless installed with `pip install -r requirements.txt`
* Listing the build tools required in `pyproject.toml`, `setup.cfg` is sort of the thing now as it lets `pip` know what it needs to build a project, i.e. that it needs `setuptools`.

Python packaging is a bit too fast moving ...